applications/openshift/networking/routes_rate_limit/tests/ocp4/e2e.yml

```diff
@@ -1,2 +1,2 @@
 ---
-default_result: PASS
+default_result: FAIL
```

Comment:
Why is this set to FAIL now?

is the cluster not passing by default? we could create a manual remediation that makes this pass. Here's an example: https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/networking/configure_network_policies_namespaces/tests/ocp4/e2e-remediation.sh

In that case, there are no networkpolicies in the cluster, so we create a new non-control plane namespace and create a network policy for it to make it pass.

Reply:
Sorry I forgot to update this, it is pass by default

Reply:
There is `test-route`, from "namespace": `"e2e-test-authentication-operator-kmtcm"`, cause this rule to fail